crates/flipperzero/src/lib.rs

```diff
@@ -1,10 +1,14 @@
 //! High-level bindings for the Flipper Zero.
-
```

Comment:
Please leave this space in (between the documentation and `#![no_std]` line). Note that `//!` is syntactic sugar for `#![doc]`, so the latter should not be treated as part of the module-level block of attributes.

Reply:
I also think that the space fits better, but for some reasong `cargo fmt` prefers to remove the blank line.

Reply:
Hmm. What if you add a `//!` before the space? That should be valid documentation, and should prevent rustfmt from joining the two.

Reply:
Done